cli/cmd/testdata/stat_one_ts_output_json.golden

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "namespace": "default",
+    "kind": "trafficsplit",
+    "name": "foo-split",
+    "success": 1,
+    "rps": 2.05,
+    "latency_ms_p50": 123,
+    "latency_ms_p95": 123,
+    "latency_ms_p99": 123,
+    "traffic_split": {
+      "apex": "apex_name",
+      "leaf": "service-1",
+      "weight": "900m"
+    }
+  },
+  {
+    "namespace": "default",
+    "kind": "trafficsplit",
+    "name": "foo-split",
+    "success": 1,
+    "rps": 2.05,
+    "latency_ms_p50": 123,
+    "latency_ms_p95": 123,
+    "latency_ms_p99": 123,
+    "traffic_split": {
```

Comment:
The structure of the json output is a bit weird.  Each object corresponds to a row from the tabular output, i.e. a leaf of a traffic split.  But the `kind` of the object is "trafficsplit".  And the leaf's name is in a sub-object called "traffic_split".  

I'm not sure there's an obvious way to make this better, though.  And since this is the json output, it being clear is less important; all the right data is here.  I just wanted to note that it's a bit weird.

Reply:
Agreed re: JSON structure. I initially considered creating a special `printTrafficSplitJSON` function. However, since `stat` supports an `all` resource type, meaning we return JSON output for multiple types of resources, modifying the JSON structure for one resource type and not others seemed potentially confusing.

Reply:
That's a good point, it would be simpler to retrieve info from the object if it's all one level. 👍